api-common/src/main/java/org/openmrs/module/metadatasharing/api/impl/MetadataSharingServiceImpl.java

```diff
@@ -211,7 +211,7 @@ protected Object task() {
 				return null;
 			}
 			
-		}.executeInManualFlushMode();
+		}.executeInAutoFlushMode();
```

Comment:
When we run this in AutoFlushMode we get the following error: https://gist.github.com/wikumChamith/1a1c66da05937e3883d0f7c922871218

This might be caused because we have added Transactional here: https://github.com/openmrs/openmrs-core/blob/889ccc1cf4d13b4e09ae9ff3ea6713cb685b3cef/api/src/main/java/org/openmrs/api/db/hibernate/HibernateContextDAO.java#L319-L322

I tested this and didn't notice any problems. I think there is a chance this could affect the performance slightly.